Dockerfile

```diff
@@ -43,8 +43,8 @@ RUN --mount=type=secret,id=aws,target=/root/.aws/credentials \
     export ${BUILD_ENVS?};  \
     if [ -z "$SCCACHE_ENDPOINT" ]; then unset SCCACHE_ENDPOINT; fi; \
     cargo build --manifest-path bin/Cargo.toml --no-default-features ${FEATURES} --release $TARGET_ARG && \
-    strip ./target/${TARGET}/release/logdna-agent && \
-    cp ./target/${TARGET}/release/logdna-agent /logdna-agent && \
+    llvm-strip ./target/${TARGET}/release/logdna-agent; \
```

Comment:
Curious why the change from `strip` to `llvm-strip`?

Reply:
That's a leftover, from the rest of the changes this was based on, will revert the non-essential stuff